debian-live/config/package-lists/hooks.chroot

```diff
@@ -0,0 +1,2 @@
+curl
```

Comment:
It has to be `hooks.list.chroot`, or?
- https://github.com/batchworksde/iksdp_desktop/tree/f22828eae32fd7fbda0a5c812fded6726dcf1d44/debian-live/config/package-lists